src/App.tsx

```diff
@@ -1,24 +1,281 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import * as postService from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10876;
+const emptyTodo = {
+  id: 0,
+  title: '',
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<string>('all');
+  const [isHideError, setIsHideError] = useState<boolean>(true);
+  const [error, setError] = useState<string>('');
+  const [query, setQuery] = useState<string>('');
+  const [tempTodo, setTempTodo]
+    = useState<{ id: number, title: string }>(emptyTodo);
```

Comment:
```suggestion
  const [tempTodo, setTempTodo] = useState<Todo>(emptyTodo);
```
Use a type for todo instead of hardcoded object